lambda_function.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+import urllib3
+import boto3
+from urllib3.exceptions import ConnectTimeoutError
+from boto3 import resource
+from botocore.exceptions import ClientError
+from boto3.dynamodb.conditions import Key, Attr
+
+http = urllib3.PoolManager()
+client = boto3.client('events', region_name="eu-west-2")
+
+
+def get_item_with_id(table, primary_key, primary_key_value):
+    response = table.query(KeyConditionExpression=Key(primary_key).eq(primary_key_value))
+    items = response["Items"]
+
+    if len(items) == 0:
+        raise Exception(f"{len(items)} items have the '{primary_key}' primary key of '{primary_key_value}'")
+    else:
+        return items[0]
+
+
+def add_true_to_ingest_cc_attribute(table, primary_key, sort_key, item_with_id):
+    primary_key_value = item_with_id[primary_key]
+    sort_key_value = item_with_id[sort_key]
+
+    table.update_item(
+        Key={primary_key: primary_key_value, sort_key: sort_key_value},
+        UpdateExpression="SET ingested_CC = :ingestedCCValue",
+        ExpressionAttributeValues={":ingestedCCValue": "true"},
+        ConditionExpression=f"attribute_exists({primary_key})"
+    )
+
+
+def get_message_from_json_event(event):
+    return json.loads(event['Records'][0]['Sns']['Message'])
```

Comment:
Please can we iterate over `Records` rather than pick the first one as this would be a prime Lambda to increase efficiency by passing a larger batch of messages.